lte/protos/session_manager.proto

```diff
@@ -787,6 +803,23 @@ message QosRules {
    uint32 qos_flow_identifier = 7;
 }
 
+message QosPolicy {
+   enum PolicyState {
+	INSTALL=0;
+	PENDING=1;
+	REJECT=2;
+   }
+   PolicyState policy_state=1;
+   uint32 version=2;
+   enum PolicyAction {
+        ADD=0;
+        DEL=1;
+        MOD=2;
```

Comment:
Now we are supporting Policy ADD and DEL only for Network Initiated Session Modification case right ? So, for ``` MOD ``` can be used for feature?

Reply:
Yes Currently we are supporting `ADD` and `DEL` policies for Network Initiated Session Modification, `MOD` is for future purpose.